cliquet/resource.py

```diff
@@ -541,10 +551,16 @@ def apply_changes(self, record, changes):
             for field, error in e.asdict().items():
                 raise_invalid(self.request, name=field, description=error)
 
-    def postprocess(self, result):
+    def postprocess(self, result, action=ACTION_READ):
         body = {
             'data': result
         }
+
+        if action != ACTION_READ:
+            # the collection has been changed
+            event = CollectionChanged(action, self, self.request)
+            self.request.registry.notify(event)
```

Comment:
I think we should notify the even only if the configuration state that we want to receive notification for this kind of even on this kind of resource.